mmdeploy/codebase/mmpose/deploy/pose_detection.py

```diff
@@ -179,6 +180,41 @@ def build_backend_model(
             **kwargs)
         return model.eval().to(self.device)
 
+    def build_pytorch_model(self,
+                            model_checkpoint: Optional[str] = None,
+                            cfg_options: Optional[Dict] = None,
+                            **kwargs) -> torch.nn.Module:
+        """Initialize torch model and switch to deploy mode.
+
+        Args:
+            model_checkpoint (str): The checkpoint file of torch model,
+                defaults to `None`.
+            cfg_options (dict): Optional config key-pair parameters.
+
+        Returns:
+            nn.Module: An initialized torch model generated by other OpenMMLab
+                codebases.
+        """
+        # Initialize the PyTorch model using parent class method
+        torch_model = super().build_pytorch_model(model_checkpoint,
+                                                  cfg_options, **kwargs)
+
+        # Check if called from torch2onnx within 'apis/pytorch2onnx.py'
+        callers = inspect.stack()
```

Comment:
we can use `torch.onnx.is_in_onnx_export()` to check if it is in onnx exportation phase.

Reply:
But the result of `torch.onnx.is_in_onnx_export()` is `False` here

Reply:
same usage found in here. Could do in this function
https://github.com/open-mmlab/mmdeploy/blob/660af62b945342276a6c813250a6fa4fcf5ff330/mmdeploy/codebase/base/task.py#L126-L130

Reply:
Thanks. I have moved `switch_to_deploy` to this method